schemas/manifest-schema.json

```diff
@@ -0,0 +1,376 @@
+        },
+        "url": {
+          "title": "package url",
+          "description": "A package url\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#package.url",
+          "type": "string"
+        },
+        "doc": {
+          "title": "package doc",
+          "description": "A package doc\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#package.docs",
+          "type": "string"
+        }
+      },
+      "required": ["name", "version"],
+      "additionalProperties": false
+    },
+    "imports": {
+      "title": "imports",
+      "description": "An import\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#imports",
+      "type": "object",
+      "propertyNames": {
+        "pattern": "^[A-Za-z_][A-Za-z0-9_]*$"
+      },
+      "additionalProperties": {
+        "type": "string"
+      },
+      "minProperties": 1
+    },
+    "protobuf": {
+      "title": "protobuf",
+      "description": "A protobuf\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#protobuf",
+      "type": "object",
+      "properties": {
+        "files": {
+          "type": "array",
+          "items": {
+            "title": "protobuf file",
+            "descriptions": "a file",
+            "type": "string",
+            "pattern": "(.proto)$"
+          },
+          "minItems": 1,
+          "unevaluatedItems": false
+        },
+        "importPaths": {
+          "type": "array",
+          "prefixFiles": [
+            {
+              "title": "protobuf prefixFile",
+              "descriptions": "a prefixFile",
+              "type":  "string"
+            }
+          ],
+          "minItems": 1
+        }
+      },
+      "required": ["files", "importPaths"],
+      "unevaluatedProperties": false
+    },
+    "binaries" : {
+      "title": "binaries",
+      "description": "A binary\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#binaries",
+      "type": "object",
+      "properties": {
+        "default" : {
+          "title": "default binary",
+          "description": "the default binary",
+          "type": "object",
+          "properties": {
+            "type": {
+              "title": "binary type",
+              "description": "A binary type\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#binaries-name-.type",
+              "enum": ["wasm/rust-v1"]
+            },
+            "file": {
+              "title": "binary file",
+              "description": "A binary file\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#binaries-name-.file",
+              "type": "string"
+            },
+            "protoPackageMapping" : {
+              "title": "binary protoPackageMapping",
+              "description": "a protoPackageMapping",
+              "type": "object",
+              "propertyNames": {
+                "pattern" : {
+                  "type": "string"
+                }
+              },
```

Comment:
Don't you need `additionalProperties` here to define what is the type of the additional property?

Reply:
it is done through unevaluatedProperties https://json-schema.org/understanding-json-schema/reference/object.html#unevaluated-properties

Reply:
I failed to see how `unevaluatedProperties` is used here to specify the `type` of the additional properties, maybe I'm missing something.